source/documentation/upgrade_guide/topics/Installing_Virtual_Product_Hypervisors_on_local_RHEL.adoc

```diff
@@ -0,0 +1,88 @@
+[id="Installing_virtual_product_hypervisors_on_local_RHEL_{context}"]
+= Installing {virt-product-shortname} hypervisors from a local_RHEL repository
```

Comment:
"local_RHEL" is misleading for me here, I would change that to "local_RHVH" repository

Reply:
ok, rereading for 3rd time, local_RHEL makes also sense if machine which is providing repos is RHEL, maybe skip _RHEL _RHVH altogether and keep only "from local repository" ?

Reply:
@pavolbrilla Thank you for reading this so carefully -- I will make the change. Is line 4 ("If your system...(CDN)" ) OK?

Reply:
yes line4 is fine